electrum/wallet.py

```diff
@@ -2459,6 +2461,180 @@ def get_tx_height(self, tx_hash: str) -> TxMinedInfo:
                 # local transaction
                 return TxMinedInfo(height=TX_HEIGHT_LOCAL, conf=0)
 
+    def get_addr_io(self, address):
+        with self.lock, self.transaction_lock:
+            h = self.get_address_history(address)
+            received = {}
+            sent = {}
+            alert_incoming = {}
+            alert_outgoing = {}
+            for tx_hash, height, tx_type, *__ in h:
+                l = self.db.get_txo_addr(tx_hash, address)
+                for n, v, is_cb in l:
+                    _key = tx_hash + ':%d'%n
+                    _value = (height, v, is_cb)
```

Comment:
1. Small `l` letter is not good name for variable. I would move `self.db.get_txo_addr(tx_hash, address)` call into `for` loop e.g `for n, v, is_cb in self.db.get_txo_addr(tx_hash, address):` or change name for e.g `txos`
2. Why not just `key` and `value` instead of `_key`, `_value`? We can skip underscore.